pkg/apis/nats/v1alpha2/cluster.go

```diff
@@ -238,6 +238,9 @@ type AuthConfig struct {
 	// ClientsAuthTimeout is the time in seconds that the NATS server will
 	// allow to clients to send their auth credentials.
 	ClientsAuthTimeout int `json:"clientsAuthTimeout,omitempty"`
+
+	// EnableTLSAuth toggles the verify and map.
+	EnableTLSAuth bool `json:"enableTLSAuth,omitempty"`
```

Comment:
imo we should keep this inline with NATS server config nomenclature.  Maybe TLSVerifyAndMap?  Wdyt?

Reply:
sounds good to me! I will update to use that instead